models/dates.js

```diff
@@ -0,0 +1,47 @@
+function getDateRange(userEndDate) {
```

Comment:
these are util functions.
models are files where we describe the schema type. ex: expected payload attributes for POST request.
create a separate folder called utils and move this dateUtils.js there